nestkernel/node.h

```diff
@@ -282,33 +282,41 @@ class Node
   }
 
   /**
-   * Bring the node from state $t$ to $t+n*dt$.
+   * Advance the state of the node forward in time by one ``min_delay``
+   * interval. Precondition: state of the node corresponds to the time
+   * ``origin``. Postcondition: state of the node corresponds to the time
+   * ``origin + min_delay``.
```

Comment:
Doxygen comments should start with a single line with a brief description that can, e.g., be included in a table of classes. And update() does not strictly advance through one min_delay interval: If Simulate is called with a time that is not a multiple of min_delay, we will stop inside a min_delay slice.

```suggestion
   * Advance the state of the node in time through the given interval.
   *
   * This method advances the state of the node through the interval 
   * ``(origin+from, origin+to]``, which is at most ``min_delay`` long.
   *
   * - Precondition: State of the node corresponds to the time ``origin+from``.
   * - Postcondition: State of the node corresponds to the time ``origin+to``.
```